src/components/shared/SampleTooltipContents.js

```diff
@@ -134,21 +132,10 @@ export class SampleTooltipContents extends React.PureComponent<Props> {
     let hasStack = false;
     let formattedSampleTime = null;
     if (sampleIndex !== null) {
-      const { samples, stackTable } = rangeFilteredThread;
+      const { samples } = rangeFilteredThread;
       const sampleTime = samples.time[sampleIndex];
-      const stackIndex = samples.stack[sampleIndex];
-      const hasSamples = samples.length > 0 && stackTable.length > 1;
-
-      if (hasSamples) {
-        const stack = getFuncNamesAndOriginsForPath(
```

Comment:
Ah, I was going to ask about this usage of `getFuncNamesAndOriginsForPath` and whether if it can be removed. But apparently you already did in the second commit, nice :)